core/src/main.rs

```diff
@@ -67,8 +67,16 @@ pub type BlockDuration = u64;
 /// The epoch counter
 pub type EpochID = u32;
 
+const ORACLE_VERSION: &str = concat!(
```

Comment:
I think `APP_VERSION` or even `VERSION` would be a better name since we don't have any other versions.